awstest/modules/sqs/module.ftl

```diff
@@ -0,0 +1,73 @@
+[#ftl]
+
+[@addModule
+    name="sqs"
+    description="Testing module for the aws sqs component"
+    provider=AWSTEST_PROVIDER
+    properties=[]
+/]
+
+[#macro awstest_module_sqs  ]
+
+    [#-- Base sqs --]
+    [@loadModule
+        blueprint={
+            "Tiers" : {
+                "app" : {
+                    "Components" : {
+                        "sqsbase" : {
+                            "sqs" : {
+                                "Profiles" : {
+                                    "Testing" : [ "sqsbase" ]
+                                },
+                                "deployment:Unit" : "aws-sqs-base",
+                                "Instances" : {
+                                    "default" : {}
+                                },
+                                "Ordering" : "FirstInFirstOut"
```

Comment:
I don't think that is required as a base level option. FIFO is an optional service

Reply:
Done